src/parser_TEST.cc

```diff
@@ -59,6 +63,31 @@ sdf::SDFPtr InitSDF()
   return sdf;
 }
 
+/////////////////////////////////////////////////
+/// Checks emitted warnings for custom/unknown elements in log file
+TEST(Parser, CustomUnknownElements)
+{
+  std::string pathBase = PROJECT_SOURCE_PATH;
+  pathBase += "/test/sdf";
+  const std::string path = pathBase +"/custom_and_unknown_elements.sdf";
+
+  sdf::SDFPtr sdf = InitSDF();
+  EXPECT_TRUE(sdf::readFile(path, sdf));
+
+  std::string pathLog = PROJECT_BINARY_DIR;
```

Comment:
I wonder if using `sdf::filesystem::append` here would fix the windows CI failure.

Reply:
Unfortunately it did not fix the Windows CI failure :confused: I'm wondering if it has something to do with `setenv` not actually being set. When using `setenv` in Windows, it should be using this right? https://github.com/osrf/sdformat/blob/00937472e6c3f70b0efee04341f48b640d882eae/test/test_config.h.in#L33-L36